server/portal/settings/settings.py

```diff
@@ -53,6 +53,7 @@ def gettext(s): return s  # noqa:E731
 
 # Custom Portal Template Assets
 PORTAL_ICON_FILENAME = settings_custom._PORTAL_ICON_FILENAME
+PORTAL_CSS_FILENAMES = settings_custom._PORTAL_CSS_FILENAMES
```

Comment:
If optional, can you provide a default here?

Reply:
I think so. Is (new) 1e8e40f the appropriate way to do that? I will deploy soon to test (on pprd utrc and pprd protx).

Reply:
Yep, thanks 👍

Reply:
The `.count` was ineffectual. I tested `|length` successfully.